source/module_hamilt_pw/hamilt_pwdft/VNL_in_pw.cpp

```diff
@@ -83,6 +83,7 @@ void pseudopot_cell_vnl::release_memory()
 // allocate vkb, PARAM.globalv.nqx, tab, tab_at
 //-----------------------------------
 void pseudopot_cell_vnl::init(const int ntype,
+                              const UnitCell& ucell,
```

Comment:
put this as the first parameter

Reply:
This parameter has already been set as the first parameter.